src/Testing/SchemaValidationTest.php

```diff
@@ -0,0 +1,73 @@
+<?php declare(strict_types=1);
+
+namespace Becklyn\PhpCs\Testing;
+
+use Doctrine\ORM\EntityManager;
+use Doctrine\ORM\Tools\SchemaValidator;
+use Doctrine\ORM\Tools\Setup;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * Validates the doctrine schema
+ */
+class SchemaValidationTest extends TestCase
+{
+    private const DB_PATH = __DIR__ . "/test_db.sqlite";
+
+    /**
+     * @inheritDoc
+     */
+    final protected function tearDown ()
+    {
+        if (\is_file(self::DB_PATH))
+        {
+            @\unlink(self::DB_PATH);
+        }
+    }
+
+    /**
+     *
+     */
+    final public function testSchemaIsValid () : void
+    {
+        $config = Setup::createAnnotationMetadataConfiguration(
+            $this->getEntityDirs(),
+            true,
+            null,
+            null,
+            false
+        );
+        $entityManager = EntityManager::create([
+            "driver" => "pdo_sqlite",
+            "path" => __DIR__ . "/db.sqlite",
```

Comment:
Is there a reason why we don't want to use SQLite with an in-memory database?

```suggestion
            "memory" => true,
```

Reply:
No specific reason, but a in-memory DB is even better.

Thanks 👍